dev/nix/flake.nix

```diff
@@ -186,16 +186,20 @@
 
             mockery = pkgs.buildGoModule rec { 
               pname = "mockery";
-              version = "2.10.0"; # upgrade this in the Makefile if upgraded here
+              version = "2.26.0"; # upgrade this in the Makefile if upgraded here
```

Comment:
How was this version not consistent with Makefile in the first place?

Reply:
Frankly not sure, I think I started working on this a while ago so the version changed while I was developing. 

Just so you are aware the versions being different is not too worrying to me because the GitHub CI tests will differ in output and fail any PR that uses the Nix dependencies incorrectly.